package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "lerna run --stream clean && lerna clean --yes",
     "fix": "lerna run --stream --concurrency 1 fix",
     "link:yarn": "lerna run --stream --concurrency 1 link:yarn",
-    "lint": "lerna run --stream --concurrency 1 lint",
+    "lint": "lerna run --stream --concurrency 9999 --no-sort lint",
```

Comment:
`lerna run --parallel lint` will do exactly same thing. We can also apply it to `clean` and `fix` (I believe).

Reply:
👍🏻